src/components/modal/modal.js

```diff
@@ -154,29 +161,34 @@ class Modal extends React.Component {
     if (!this.props.enableBackgroundUI) {
       return (
         <div
-          className="carbon-modal__background"
+          className='carbon-modal__background'
         />
       );
     }
+    return null;
   }
 
+  /* eslint-disable no-useless-return */
+
```

Comment:
Good point, thought it was complaining as there was a return with no content.

This makes more sense